thirdparty/vars.sh

```diff
@@ -390,8 +390,15 @@ DATASKETCHES_NAME="datasketches-cpp-4.0.0.tar.gz"
 DATASKETCHES_SOURCE="datasketches-cpp-4.0.0"
 DATASKETCHES_MD5SUM="724cd1df9735de2b8939d298f0d95ea2"
 
+# async-profiler
+ASYNC_PROFILER_DOWNLOAD="http://starrocks-thirdparty.oss-cn-zhangjiakou.aliyuncs.com/async-profiler-2.9-linux-x64.tar.gz"
```

Comment:
will break arm build

Reply:
suggest using github link, so maintainer can still find the original location and update if necessary.

https://github.com/async-profiler/async-profiler/releases

Reply:
done